### PR TITLE
Add promptid to every button

### PR DESCRIPTION
**Pull request by cryogen**
To make sure button clicks aren't processed if the current step does not permit it (ie, when theyve been clicked more than once)

### CODE REVIEW

File: server/game/gamepipeline.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ class GamePipeline {
     getCurrentStep() {
         var step = this.pipeline[0];
 
-        if(typeof(step) === 'function') {
+        if(typeof (step) === 'function') {
```
Comment (ystros) [inline server/game/gamepipeline.js]: Do we have eslint rules for this (if this is the formatting we want)?

File: server/game/game.js [hunk 2]
```diff
@@ -75,11 +75,11 @@ class Game extends EventEmitter {
         this.skipPhase = {};
         this.cardVisibility = new CardVisibility(this);
 
-        for(let player of details.players || []) {
+        for(let player of Object.values(details.players || {})) {
```
Comment (ystros) [inline server/game/game.js]: Seems fairly unrelated to this change?

Comment (cryogen) [reply server/game/game.js]: It is, but the game node server won’t run without this being fixed :)

File: server/game/gamepipeline.js (hunk 1 above)
Comment (cryogen) [reply server/game/gamepipeline.js]: No, and I’m not even sure it is what we want, I have too many instances of vs code with differing configs.  I will sort that out in my end

**Pull request merged**